src/main/java/no/nav/pto/veilarbportefolje/kafka/KafkaCommonConsumerService.java

```diff
@@ -3,9 +3,21 @@
 import lombok.extern.slf4j.Slf4j;
 import org.apache.kafka.clients.consumer.ConsumerRecord;
 
+import static no.nav.pto.veilarbportefolje.kafka.KafkaConfigCommon.Topic.ENDRING_PAA_OPPFOLGINGSBRUKER;
+
 @Slf4j
 public abstract class KafkaCommonConsumerService<T> {
     public void behandleKafkaRecord(ConsumerRecord<String, T> kafkaMelding) {
+        if(ENDRING_PAA_OPPFOLGINGSBRUKER.getTopicName().equals(kafkaMelding.topic())){
```

Comment:
Er dette ok? Denne topicen har fnr som key

Reply:
I wouldnt add that in KafkaCommonConsumerService, since that class is used for each kafka consumer